### PR TITLE
💄 feat: updated editor design

### PR DESCRIPTION
**Pull request by pferdlexxie**

### CODE REVIEW

File: src/lib/components/TransactionView.svelte
Comment (dprostiruk) [inline src/lib/components/TransactionView.svelte:1]: Nice aligning☺️ Tags look reliable to me😉

**Review by dprostiruk — COMMENTED** [review]
Nice job, great changes😁

File: src/lib/components/TransactionView.svelte [hunk 1]
```diff
@@ -5,62 +5,87 @@
...
             </div>
-        </div>
 
-        <div class="flex items-left">
-            <!-- Adding the 'flex' and 'items-left' classes here -->
-            <div>Date:</div>
-            <div>
+            <div class="flex items-left">
+                <!-- Adding the 'flex' and 'items-left' classes here -->
+                <!-- <div>Date:</div> -->
```
Comment (cuprum-acid) [inline src/lib/components/TransactionView.svelte:33]: Commenting this tag looks not reliable to me 🤔

File: src/lib/components/TransactionView.svelte [hunk 2]
```diff
@@ -5,62 +5,87 @@
...
+        <div class="flex flex-col w-32 text-xl text-start mt-3 mr-8">
+            <div class="mb-6">ID</div>
+            <div class="mb-6">Date</div>
+            <div class="mb-6">Amount</div>
+            <div>Category</div>
+        </div>
+        <div class="join-vertical space-y-1">
+            <div class="flex items-left">
+                <!-- Adding the 'flex' and 'items-left' classes here -->
+                <!-- <div>ID:</div> -->
```
Comment (cuprum-acid) [inline src/lib/components/TransactionView.svelte:19]: I think we need to delete this later 🤔

File: src/lib/components/TransactionView.svelte [hunk 3]
```diff
@@ -5,62 +5,87 @@
...
     </div>
+    <div class="flex max-w-lg mx-auto justify-end mx-3 pb-1">
+        <button class="btn btn-normal w-32"> Save </button>
+    </div>
 </div>
+
+<style>
+    .btn {
+        background-color: #007349;
+        color: #fefffe;
```
Comment (cuprum-acid) [inline src/lib/components/TransactionView.svelte:84]: Nice color choose! 😄

File: src/lib/components/TransactionView.svelte [hunk 4]
```diff
@@ -5,62 +5,87 @@
...
+                    </div>
                 </div>
             </div>
+            <!-- <div class="flex max-w-3xl mt-3">
+            <button class="btn btn-wide"> +Add </button>
+        </div> -->
         </div>
     </div>
+    <div class="flex max-w-lg mx-auto justify-end mx-3 pb-1">
+        <button class="btn btn-normal w-32"> Save </button>
```
Comment (cuprum-acid) [inline src/lib/components/TransactionView.svelte:77]: Good and reliable "Save" button. I like it 😍

**Review by cuprum-acid — APPROVED** [review]
Good design style! I love #FEFFFE experience ❤️

File: src/lib/components/TransactionView.svelte (hunk 1 above)
Comment (axonstan) [reply src/lib/components/TransactionView.svelte:33]: This comment also does not look reliable to me

**Review by axonstan — APPROVED** [review]

**Pull request merged**